api/controllers/category.js

```diff
@@ -0,0 +1,72 @@
+const { category } = require('../../models/')
+
+//Category
+module.exports.getAllCategory = async (req, res) => {
+  const tempCategory = await category.findAll().
```

Comment:
lambda periods shouldn't be on separate lines
```suggestion
  const tempCategory = await category.findAll()
    .catch((err) => {
```